dicom_utils/types.py

```diff
@@ -18,34 +18,134 @@
 WINDOW_WIDTH = 0x00281051
 UNKNOWN: Final = -1
 
+
 Dicom = Dataset
 DicomAttributeSequence = Sequence
 
 
+class ModalityError(Exception):
+    pass
+
+
+def get_value(dcm: Dicom, tag: Tag, default: Any) -> Any:
+    return dcm.get(tag).value if tag in dcm else default
+
+
 def get_tag_values(tags: Iterable[Tag], dcm: Dicom) -> Dict[Tag, Any]:
-    tags = {tag: dcm[tag].value if tag in dcm else None for tag in tags}
+    tags = {tag: get_value(dcm, tag, None) for tag in tags}
     return tags
 
 
-class SimpleImageType(Enum):
+# TODO: these would be better in dicom.py, but circular import issues
+def view_code_iterator(dcm: Dataset) -> Iterator[Dataset]:
```

Comment:
This is personal preference, but I like to start function names with a verb (e.g. `get_`, `check_`, etc.). What would you think of the name `iterate_view_code` instead of `view_code_iterator`? Same question regarding the next function, `view_modifier_code_iterator`.